daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/outputValueCalc.tdml

```diff
@@ -708,4 +742,63 @@
 	    </tdml:errors>
 
 	</tdml:unparserTestCase>
+
+    <tdml:unparserTestCase name="refSimpleTypeElemWithOvc"
```

Comment:
Why are these unparserTestCases ? Was this bug particular to unparsing? 
You can convert these to parserTestCase I think.

Reply:
I went with unparserTestCase since we're dealing with testing outputValueCalcs and I only cared about the results of unparsing the provided infoset (i.e roundTrip='false') and triggering the OVC code path, and not the results of parsing. Also for consistency,  since the only tests in the file are unparserTestCase.

Reply:
I thought OVCs were ignored during parsing.

Reply:
My mistake. I was reading these as InputValueCalc.... maybe I need new glasses ?

+1.